### ISSUE TITLE
[Bug]: typescript wrong type for Grammarly

### ISSUE THREAD

**Issue by aliakbarazizi** [grammarly/grammarly-for-developers#632]
### What package(s) are you seeing the problem in?

@grammarly/editor-sdk-react

### SDK version

2.3.9

### Browser(s)

Firefox, Chrome

### Operating system(s)

Windows

### Steps to reproduce

Create a project in typescript and add this

```
<Grammarly clientId={grammarly}>
     rest of code...
</Grammarly>
```

### What happened?

The typescript error

```
Type '{ children: Element; clientId: string; }' is not assignable to type 'IntrinsicAttributes & GrammarlyProps'.
  Property 'children' does not exist on type 'IntrinsicAttributes & GrammarlyProps'
```

### What did you expect to happen?

No error

**Comment by cwatkins** [comment]
Hi @aliakbarazizi, 

Welcome to the community, and thank you for reporting this issue.

I get the same message when using `@grammarly/editor-sdk-react` with TypeScript. I'll bring this up with the team.

**Comment by wachunga** [comment]
Thanks for reporting this! It'll be fixed in 2.4.0, which should land later today or tomorrow.

**Comment by cwatkins** [comment]
Hi @aliakbarazizi,

2.4.0 has been released, and I can confirm that the TypeScript error no longer shows up on the `Grammarly` provider. You can see the same in [this demo using version 2.4.0](https://codesandbox.io/s/async-breeze-ojxm5c?file=/src/Editors.tsx) written in TypeScript.  

I'm closing this bug for now, but if you have the error even after upgrading the Text Editor SDK to version 2.4.0, please let us know.
